src/glmfit.jl

```diff
@@ -233,10 +233,13 @@ function _fit!(m::AbstractGLM, verbose::Bool, maxIter::Integer, minStepFac::Real
         installbeta!(p, f)
         crit = (devold - dev)/dev
         verbose && println("$i: $dev, $crit")
-        if crit < convTol
+        if crit < convTol || dev == 0
             cvg = true
             break
         end
+        if !isfinite(crit)
+            error("convergence criterion is $crit") # maybe change ConvergenceException to allow for custom error
+        end
```

Comment:
@ararslan That's not really practical when you have 100 or more coefficients, which has happened to me many times (with log-mutiplicative models). Anyway it's always better to be able to access the vector object rather than just a string (with precision loss, etc.).

But indeed if that can't happen then there's no hurry to add this (though in general adding the criterion to `ConvergenceException` wouldn't be a bad idea).